src/Component/Filter/ComparisonFilter/ComparisonFilter.tsx

```diff
@@ -273,15 +273,15 @@ class ComparisonFilterUi extends React.Component<ComparisonFilterProps, Comparis
       attribute: isValid ? 'success' : 'error'
     };
 
-    if (_isFunction(onValidationChanged)) {
-      onValidationChanged(validationStateNew);
-    }
-
     onFilterChange(filter);
     this.setState({
       filter,
       validateStatus: validationStateNew
-    });
+    },            () => {
```

Comment:
Is this the correct indentation?

Reply:
According to the current `tslint.config`, yes

Reply:
OK, nice :wink: